### PR TITLE
Add filter for currency

### PR DESCRIPTION
**Pull request by zetske**

### CODE REVIEW

**Review by xdemocle — DISMISSED** [review]

**Comment by xdemocle** [comment]
commits are still missing verification

**Review by xdemocle — DISMISSED** [review]

File: src/modules/land-works/views/explore-view/index.tsx [hunk 1]
```diff
@@ -46,11 +50,21 @@ const ExploreView: React.FC = () => {
...
+  const [searchQuery, setSearchQuery] = useState(query || '');
+
   const [claimButtonDisabled, setClaimButtonDisabled] = useState(false);
   const [loading, setLoading] = useState(false);
   const [showOnlyOwner, setShowOnlyOwner] = useState(false);
   const [showOnlyAvailable, setShowOnlyAvailable] = useState(false);
-  const [, setCurrency] = useState(tokenOptions[0]);
+
+  const [paymentTokens, setPaymentTokens] = useState([] as PaymentToken[]);
+  const [paymentToken, setPaymentToken] = useState(DEFAULT_TOKEN_ADDRESS);
```
Comment (xdemocle) [inline src/modules/land-works/views/explore-view/index.tsx:66]: this setPaymentToken and setPaymentTokens and paymentTokens and paymentToken are confusing var names.

consider change paymentToken to paymentTokenSelected for example.

File: src/modules/land-works/views/explore-view/index.tsx [hunk 2]
```diff
@@ -105,7 +137,21 @@ const ExploreView: React.FC = () => {
...
+    }
+
+    return lands.filter((land) => {
+      const landName = land.name.toLowerCase();
+      const landOwner = land.decentralandData?.asset?.consumer?.id.toLowerCase();
+      return landName.includes(query) || landOwner?.includes(query);
+    });
+  };
+
+  const filteredLands = filterLandsByQuery(lands, searchQuery);
```
Comment (xdemocle) [inline src/modules/land-works/views/explore-view/index.tsx:154]: this is not okay. in my explore view im using only one lands and only setLands.

Lands var names `lands` updates based on the AssetsEntity[] we pass.

So, I think is better you wait my imminent PR of today to be merged... so you can take it from there.

**Review by xdemocle — CHANGES_REQUESTED** [review]

File: src/modules/land-works/views/explore-view/index.tsx (hunk 1 above)
Comment (zetske) [reply src/modules/land-works/views/explore-view/index.tsx:66]: This is the variable names used in `list-property-view` but I'll update it

File: src/modules/land-works/components/lands-search/index.tsx [hunk 3]
```diff
@@ -9,8 +9,12 @@ interface SearchQuery {
 
 export const SearchBar: React.FC<SearchQuery> = ({ searchQuery, setSearchQuery }) => (
   <div className="wrapper lands-search">
+    <Search className="search-icon" />
```
Comment (xdemocle) [inline src/modules/land-works/components/lands-search/index.tsx:12]: is also a good habit using suffix Icon, ex. SearchIcon in declaring the var name

**Pull request closed without merging**